dataland-frontend/tests/e2e/specs/framework-view-and-upload-journey/UploadFrameworkDataJourney.ts

```diff
@@ -210,6 +210,7 @@ describe("As a user, I expect the dataset upload process to behave as I expect",
           " and verify pages and elements",
         function () {
           cy.visitAndCheckAppMount("/companies");
+          cy.wait(10000);
```

Comment:
should this stay?

Reply:
Will check if this can be removed once the e2e tests are working again

Reply:
Tests worked without it, so i will delete the wait statement.

Reply:
ok